test/pytorch_mnist/combiner/run.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+fedn run fedavg -d discovery -p 8080 -t e9a3cb4c5eaff546eec33ff68a7fbe232b68a192 -h combiner -i 12080 -n combiner
```

Comment:
Please ask Morgan for help how to remove the git-lfs files, and instead provide detailed instructions for how to obtain the files and where to stage them in the example dir, in the example readme.